documentation/src/main/markdown/packagemgrs/golang.md

```diff
@@ -37,6 +38,13 @@ the [go mod why documentation](https://go.dev/ref/mod#go-mod-why) for additional
 
 [detect_product_short] does not exclude any dependencies from the BOM by default.
 
+## Go Mod File (GO_MOD) detector
+
+* Discovers dependencies of Go projects that uses Go Mod package manager.
```

Comment:
Suggestion:
* Discovers dependencies of Go projects that use Go Module package manager.